test/ApiEndPointsTest.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+namespace app\tests;
+
+use GuzzleHttp\Client;
+
+class ApiEndPointsTest extends \PHPUnit_Framework_TestCase
+{
+    public $client;
+
+    public function setUp()
+    {
+        $this->client = new Client(['base_url' => 'http://bd-naijaemoji.herokuapp.com/']);
```

Comment:
**Testing - Nitpick**
Will be good to have the application self-contained  and testable offline.